add-chain/testdata/superchain/configs/sepolia/expected_plasma.yaml

```diff
@@ -0,0 +1,24 @@
+name: awesomechain_plasma
+chain_id: 42069
+public_rpc: http://awe.some.rpc
+sequencer_rpc: http://awe.some.seq.rpc
+explorer: https://awesomescan.org
+
+superchain_level: 2
+
+batch_inbox_addr: 0xff00000000000000000000000000000000042069
+
+genesis:
+  l1:
+    hash: 0x1fc104fe244042459314bc1d51a230320c64a3d47cd06b7998aa3f57b9b0fc79
+    number: 5753349
+  l2:
+    hash: 0x0ceefff2113580e4d665c324661e4982a824025bf97285cf09a52715b6f89083
+    number: 0
+  l2_time: 1713792864 # Mon 22 Apr 2024 13:34:24 UTC
+
+plasma:
```

Comment:
@trianglesphere this additional `plasma` field and the fields nested underneath it are the only differences between this .yaml file produced by a vanilla chain and a plasma chain. Is that correct?

Reply:
Yes that is correct. Do we want to duplicate the `da_challenge_contract_address`?

Reply:
Are you saying we should de-duplicate by removing the address from this .yaml file and just rely on the address being in the [.json file](https://github.com/ethereum-optimism/superchain-registry/blob/7f3dc3e4d185465712835d8505e20bc6f7bcc17c/add-chain/testdata/superchain/extra/addresses/sepolia/expected_plasma.json#L4) along with the other contract addresses?

Reply:
Yea, I think so. We do that with the other addresses